package.json

```diff
@@ -12,8 +12,8 @@
     "mocha": "mocha test/**/*.js --exit",
     "mocha:web": "mocha test/web/*js --exit",
     "mocha:node": "mocha test/node/*js --exit",
-    "build": "webpack --mode=production --config webpack.production.js",
-    "serve": "webpack-dev-server --mode=development --config webpack.development.js",
+    "build": "NODE_OPTIONS=--openssl-legacy-provider webpack --mode=production --config webpack.production.js",
```

Comment:
@tranjennie what are these changes for in project.json? Is this just for your local dev-env?

Reply:
@beefo I was having issues with serving and building with node v18.4.0. I'm now using node v16.16.0 and have no issues with serving and building. I will remove these changes.